backend/src/akvo/lumen/import.clj

```diff
@@ -6,7 +6,9 @@
              [akvo.lumen.util :as util]
              [cheshire.core :as json]
              [clojure.java.jdbc :as jdbc]
-             [hugsql.core :as hugsql]))
+             [hugsql.core :as hugsql])
+  (:import [org.postgresql PGConnection]
+           [org.postgresql.util PGobject]))
```

Comment:
Shouldn't we import `org.postgis.Polygon` and `org.postgis.MultiPolygon` instead of `PGConnection` ?

Reply:
👍 fixed